### DISCUSSION TITLE
Communication Solution

### DISCUSSION THREAD

**Discussion by MKr4zy** [RainerGa/CCMT#22] [Ideas]
we need some sort of Program/solution, where we can discuss problems or brainstorm or whatever.
Here are some Ideas:

[Rocket.Chat](https://github.com/RocketChat/Rocket.Chat)
[Gitter](https://gitter.im)
[slack](https://slack.com/intl/de-de/)

**Comment by RainerGa** [comment]
Good! Please send me the tool we will use.
